providers/atlassian/resources/atlassian_jira.go

```diff
@@ -226,13 +226,28 @@ func (a *mqlAtlassianJira) issues() ([]interface{}, error) {
 				return nil, err
 			}
 
+			creator := issue.Fields.Creator
+			mqlAtlassianJiraUser, err := CreateResource(a.MqlRuntime, "atlassian.jira.user",
+				map[string]*llx.RawData{
+					"id":      llx.StringData(creator.AccountID),
+					"name":    llx.StringData(creator.DisplayName),
+					"type":    llx.StringData(creator.AccountType),
+					"picture": llx.StringData(creator.AvatarUrls.One6X16),
+				})
+			if err != nil {
+				return nil, err
```

Comment:
I've no clue about Jira, so perhaps a dump question:
Could someone be permitted to view the issues but not the users?